src/mono/wasm/debugger/tests/debugger-test/debugger-main.js

```diff
@@ -22,7 +22,7 @@ try {
         },
     }));
     App.runtime = runtime;
-    App.init()
+    await App.init();
```

Comment:
@radical FYI app will be ready only after this is resolved

Reply:
Related to https://github.com/dotnet/runtime/issues/73528

Reply:
I think @thaystg  wanted to move `mono_wasm_runtime_ready` console log line into the unit test. That would be right place and right signal